core/src/main/resources/error/error-classes.json

```diff
@@ -1719,7 +1708,7 @@
   },
   "WRONG_NUM_ARGS" : {
     "message" : [
-      "Invalid number of arguments for the function <functionName>."
+      "The <functionName> requires <expectedNum> parameters but the actual number is <actualNum>."
     ],
     "subClass" : {
       "WITHOUT_SUGGESTION" : {
```

Comment:
I think the error class name `WRONG_ NUM_ ARGS.WITHOUT_ SUGGESTION` is not very accurate after this pr because `expectedNum` and `actualNum` have been pointed out in the message。

Reply:
Honestly, I think `expectedNum` and `actualNum` are sort of basic information rather than a suggestion.

That's why I put them into main error class instead of sub error classes.